### PR TITLE
Finished section Testing JavaScript Execution in WebViews

### PR DESCRIPTION
**Pull request by xep624**
Please review section Testing JavaScript Execution in WebViews.

### CODE REVIEW

File: Document/0x06h-Testing-Platform-Interaction.md [hunk 1]
```diff
@@ -169,41 +169,86 @@
...
+
+* Stored Cross-Site Scripting (XSS) vulnerability in an endpoint, where the exploit will be sent to the WebView of the Mobile App when navigating to the vulnerable function.
+* Man-in-the-middle (MITM) position by an attacker where he is able to tamper the response by injecting JavaScript.
 
 #### Remediation
 
--- TODO [Describe the best practices that developers should follow to prevent this issue "Testing JavaScript Execution in WebViews".] --
+The UIWebView should be avoided and WKWebView used instead. JavaScript is enabled by default in a WKWebView and should be disabled if not needed. This reduces the attack surface and potential threats to the application. If JavaScript is needed it should be ensured:
+
+In order to address these attack vectors, the outcome of the following checks should be verified:
```
Comment (sushi2k) [inline Document/0x06h-Testing-Platform-Interaction.md]: I would remove this sentence as already the sentence before "If JavaScript is needed it should be ensured:" is referring to the bullet points below.

File: Document/0x06h-Testing-Platform-Interaction.md [hunk 2]
```diff
@@ -169,41 +169,86 @@
...
+* that the HTTPS communication need to be implemented according to the best practices to avoid MITM attacks (see "Testing Network Communication").
+
 
 #### References
 
-##### OWASP Mobile Top 10 2016
-* M7 - Client Code Quality - https://www.owasp.org/index.php/Mobile_Top_10_2016-M7-Poor_Code_Quality
+##### OWASP Mobile Top 10 2014
+
+* M7 - Client Side Injection
```
Comment (sushi2k) [inline Document/0x06h-Testing-Platform-Interaction.md]: Can you please link to the Mobile Top 10 2016?
* M7 - Client Code Quality - https://www.owasp.org/index.php/Mobile_Top_10_2016-M7-Poor_Code_Quality

File: Document/0x06h-Testing-Platform-Interaction.md (hunk 1 above)
Comment (xep624) [inline Document/0x06h-Testing-Platform-Interaction.md]: Ups... you're right :) Thanks!

File: Document/0x06h-Testing-Platform-Interaction.md (hunk 2 above)
Comment (xep624) [inline Document/0x06h-Testing-Platform-Interaction.md]: Done. Thanks!

**Pull request merged**